src/match_traildb.c

```diff
@@ -551,6 +552,9 @@ int run_groupby_query2(char **traildb_paths, int num_paths, groupby_info_t *gi,
                 id = *(__uint128_t *)cookie;
             }
 
+            if (exclude_set && exclude_set_contains(exclude_set, cookie))
+                continue; // If the uuid is in the exclude_set skip its trail
```

Comment:
Comment is a bit obvious

Reply:
Yeah, just wanted to point out what the `continue` implication was there.